### DISCUSSION TITLE
Is "multiple emails" feature possible?

### DISCUSSION THREAD

**Discussion by texpert** [janko/rodauth-rails#312] [General]
Hello, @janko!

I'm trying to integrate rodauth-rails in a new application, but I'd like to have a separate `emails` table and Email model to allow the users to have multiple emails. Like, for example, in https://github.com/allenwq/devise-multi_email

Is it somehow possible via some hooks/settings?

**Comment by janko** [comment]
It's probably possible, you'd just have to override lots of Rodauth methods to achieve it (similar to how the `device-multi_email` gem overrides a lot of Devise). Moving to a discussion, since it's not a bug report.
